docs/labs.md

```diff
@@ -148,4 +148,9 @@ development will provide opportunities for feedback.
 Maximised widgets provide a room layout in which a widget is (temporarily) the primary focus of the room. The whole chat area is then used for the widget. The chat is moved into the right panel.
 
 Note that this feature is currently under active development and therefore is
-entirely incomplete and may not work at all - it is not recommended for general use at this time.
\ No newline at end of file
+entirely incomplete and may not work at all - it is not recommended for general use at this time.
+
+## Metaspaces (`feature_spaces_metaspaces`) [In Development]
+
+Metaspaces are automatically populated spaces you can enable in your Space panel.
+By default, you'll have Home or All rooms, but you can opt in to a People, Favourites, and Orphans metaspace too.
```

Comment:
```suggestion
By default, you'll have Home or All rooms, but you can opt in to a People, Favourites, and Other Rooms metaspace too.
```

I assume that via [`2c13035` (#7077)](https://github.com/matrix-org/matrix-react-sdk/pull/7077/commits/2c1303579ff324595a38d0ab301ccb1531cd1daf)(?)

Reply:
Thanks!